cram_pr2/cram_pr2_plan_transformation/cram-pr2-plan-transformation.asd

```diff
@@ -0,0 +1,99 @@
+;;; IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+;;; ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+;;; LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+;;; CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+;;; SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+;;; INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+;;; CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+;;; ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+;;; POSSIBILITY OF SUCH DAMAGE.
+
+(defsystem cram-pr2-plan-transformation
+  :author "artnie"
+  :license "BSD"
+  
+  :depends-on (roslisp-utilities ; for ros-init-function
+
+               cl-transforms
+               cl-transforms-stamped
+               cl-tf
+               cram-tf
+
+               cram-language ; obsolete
+               cram-executive
+               cram-designators ; obsolete
+               cram-prolog
+               cram-projection
+               cram-occasions-events
+               cram-utilities ; obsolete
+
+               cram-common-failures
+               cram-mobile-pick-place-plans
+               cram-object-knowledge
+
+               cram-cloud-logger
+
+               cram-physics-utils     ; for reading "package://" paths
+               cl-bullet ; for handling BOUNDING-BOX datastructures
+               cram-bullet-reasoning ; obsolete
+               cram-bullet-reasoning-belief-state ;; using cram-bullet-reasoning
+               cram-bullet-reasoning-utilities
+               cram-btr-visibility-costmap ;
+               cram-btr-spatial-relations-costmap ;
+
+               ;; cram-semantic-map-costmap
+               cram-robot-pose-gaussian-costmap ;; contains cram-location-costmap
+               cram-occupancy-grid-costmap ;; contains cram-location-costmap
+               cram-location-costmap
+
+               cram-pr2-projection      ; for with-simulated-robot, depends on bullet-resoning
+               cram-pr2-projection-reasoning ; for projection-based reasoning
+               cram-pr2-description ;
+               cram-pr2-fetch-deliver-plans
+               cram-pr2-environment-manipulation ;
+
+               
+
+               ;; cram-robosherlock
+               ;; cram-bullet-reasoning-designators
+               ;; cram-bullet-reasoning-costmap ; not using any spatial relation cms yet
+               ;; cram-bullet-reasoning-designators ; not using visibility cm or collision checks
+               
+               )
+  :components
+  ((:module "src"
+    :components
+    ((:file "package")
+     (:file "setup" :depends-on ("package"))
+     (:file "plan-transformation" :depends-on ("package"))
+     (:file "predicates" :depends-on ("package"))
+     (:file "projection-demo" :depends-on ("package"))
+     (:file "demo" :depends-on ("package" "setup" "projection-demo"))
```

Comment:
This package should not contain any demo code or dependencies on demo stuff, like PR2 packages. Please remove the demo code and use the demo functions from ``cram_pr2_pick_place_demo`` if you need something to test on.